modules/delegates.js

```diff
@@ -203,6 +205,93 @@ __private.forge = function (cb) {
 };
 
 /**
+ * Returns the decrypted secret by deciphering encrypted secret with the key provided
+ * using aes-256-cbc algorithm.
+ * @private
+ * @param {string} encryptedSecret
+ * @param {string} key
+ * @returns {string} decryptedSecret
+ * @throws {error} if unable to decrypt using key
+ */
+__private.decryptSecret = function (encryptedSecret, key) {
+	var decipher = crypto.createDecipher('aes-256-cbc', key);
+	var decryptedSecret =	decipher.update(encryptedSecret, 'hex', 'utf8');
+	decryptedSecret += decipher.final('utf8');
+	return decryptedSecret;
+};
+
+/**
+ * Updates the forging status of an account, valid actions are enable and disable
```

Comment:
Punctuation: `Updates the forging status of an account, valid actions are enable and disable.`